themes/mixin-icons-fontawesome6/css/vendor/fontawesome-free-6.6.0-web/LICENSE.txt

```diff
@@ -0,0 +1,165 @@
+Fonticons, Inc. (https://fontawesome.com)
```

Comment:
Is it possible to load the FontAwesome files via npm to simplify future upgrades?